lib/services/local/saml_idp_service_provider.go

```diff
@@ -46,6 +46,7 @@ const (
 	samlIDPServiceProviderModifyLock    = "saml_idp_service_provider_modify_lock"
 	samlIDPServiceProviderModifyLockTTL = time.Second * 5
 	samlIDPServiceProviderMaxPageSize   = 200
+	teleportSAMLIdP                     = "TELEPORT_SAML_IDP"
```

Comment:
Nit: I think the purpose of this constant would be slightly clearer with this name.
```suggestion
	samlIDPServiceName                  = "TELEPORT_SAML_IDP"
```